cranelift/interpreter/src/value.rs

```diff
@@ -34,17 +33,6 @@ pub trait Value: Clone + From<DataValue> {
     fn min(self, other: Self) -> ValueResult<Self>;
 
     // Comparison.
-    fn eq(&self, other: &Self) -> ValueResult<bool>;
-    fn gt(&self, other: &Self) -> ValueResult<bool>;
-    fn ge(&self, other: &Self) -> ValueResult<bool> {
-        Ok(self.eq(other)? || self.gt(other)?)
-    }
-    fn lt(&self, other: &Self) -> ValueResult<bool> {
-        other.gt(self)
-    }
-    fn le(&self, other: &Self) -> ValueResult<bool> {
-        Ok(other.eq(self)? || other.gt(self)?)
-    }
```

Comment:
These needed conflicted with PartialEq/PartialOrd impls of DataValue causing ambiguity during method resolution. I took the opportunity to inline them all.